packages/lesswrong/components/users/UsersProfile.tsx

```diff
@@ -276,6 +365,9 @@ const UsersProfileFn = ({terms, slug, classes}: {
                   </DialogGroup>
                 </div>
               }
+              { currentUser && currentUser._id === user._id && <Link to="/profile/edit">
+                Edit Profile
+              </Link>}
```

Comment:
I just put this link on the profile page, and not in the username dropdown menu. For now only the user themselves can edit profile data - not sure if it would be useful for admins to be able to edit user profile data via the site.

Reply:
Oh, now I see this. It was outdated by the time I looked at the diff